ext/drreg/drreg.h

```diff
@@ -369,6 +405,17 @@ DR_EXPORT
 drreg_status_t
 drreg_init_and_fill_vector(drvector_t *vec, bool allowed);
 
+DR_EXPORT
+/**
+ * Identical to drreg_init_and_fill_vector(), but for a specific register
+ * class based on spill class \p spill_class.
```

Comment:
Best to document the size of the vector and which registers it holds for which class: or put that up by the classes (for a sep comment for each) and point there here.

Reply:
Done.